src/button.tsx

```diff
@@ -354,8 +356,11 @@ const Button = React.forwardRef<TouchableElement, ButtonProps & {type: ButtonTyp
         style: {cursor: props.fake ? 'pointer' : undefined, ...props.style},
         trackingEvent: props.trackingEvent ?? (props.trackEvent ? createDefaultTrackingEvent() : undefined),
         dataAttributes: props.dataAttributes,
+        'aria-label': props['aria-label'],
         'aria-controls': props['aria-controls'],
         'aria-expanded': props['aria-expanded'],
+        'aria-haspopup': props['aria-haspopup'],
+        'aria-busy': showSpinner ? true : undefined,
```

Comment:
the current implementation is already working with voice-over while loading state.. so... I wonder why do we need aria-busy 🤔

Reply:
If it has no effect, I'd remove it

Reply:
Agree. Removed

Reply:
Good catch, I tested a button with aria-busy to see if it was working, but I didn't notice it was the same behavior as the case when it doesn't have aria-busy.

Removed this prop :)